packages/shared/catalogs/sagas.ts

```diff
@@ -184,6 +197,26 @@ async function fetchWearablesByFilters(filters: WearablesRequestFilters, client:
   return client.fetchWearables(filters)
 }
 
+/**
+ * Fetches a single item from the Builder Server.
+ */
+async function fetchWearableByIdFromBuilder(uuidsItems: string[], identity: ExplorerIdentity): Promise<WearableV2[]> {
```

Comment:
Maybe we need to name `fetchWearableByIdFromBuilder` plural `fetchWearablesByIdFromBuilder`?

Reply:
Done!